src/engine/Actor.ts

```diff
@@ -758,7 +794,7 @@ module ex {
      * @param recurse checks whether the x/y are contained in any child actors (if they exist).
      */
     public contains(x: number, y: number, recurse: boolean = false): boolean {
-       var containment = this.getBounds().contains(new Point(x, y));
+       var containment = this.getBounds().contains(new Vector(x, y));
```

Comment:
Wonder if `contains` should accept either a Vector or (x, y) to avoid new'ing up temp Vectors.